downstream/modules/platform/proc-installing-the-ansible-builder-rpm.adoc

```diff
@@ -1,18 +1,15 @@
+//Used in downstream/titles/aap-installation-guide/platform/assembly-disconnected-installation.adoc
+
+
 [id="installing-the-ansible-builder-rpm_{context}"]
 
-= Installing the ansible-builder RPM
+= Installing the {Builder} RPM
 
 [role="_abstract"]
 
-To create custom {ExecEnvShort} images in a disconnected environment, you must have a working copy of ansible-builder installed. On a RHEL system there are several ways to do this.
 
-* Subscribe the RHEL box to a Satellite on the disconnected network.
-* Attach the {PlatformNameShort} subscription and enable the {PlatformNameShort} repo.
+On the RHEL system where custom {ExecEnvShort}s will be built, you will install the {Builder} RPM using a Satellite server that already exists in the environment. This method is preferred because the {ExecEnvShort} images can use any RHEL content from the pre-existing Satellite if required.
 
-[NOTE]
-====
-This is preferred if a Satellite exists because the {ExecEnvShort} images can use RHEL content from the Satellite if the underlying build host is registered.
-====
 
 .Prerequisites
```

Comment:
this prerequisite no longer belongs in this section, it is covered at the end of the subsequent section ("Creating the customer {ExecEnvshort} definition")